### PR TITLE
Add time machine narratives.

### PR DESCRIPTION
**Pull request by crperezt**
Please review this, @casey-collab

### CODE REVIEW

File: english/time-machine/time-machine.md [hunk 1]
```diff
@@ -4,3 +4,10 @@ There is no way back to the current time.
 So think carefully before you choose.
 
 Which part of the world, and what time you want to travel? And why?
+
+[You decide to travel to 2135 and say goodbye to the moonmen](https://www.youtube.com/watch?v=TgqiSBxvdws)
+
+[You travel to 2017 and meet with an Apple Engineer](https://www.youtube.com/watch?v=-XSC_UG5_kU)
+
+[You turn back](english/marshmallow.md)
```
Comment (casey-collab) [inline english/time-machine/time-machine.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by crperezt** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :thumbsup:

**Pull request merged**